website/src/styles/poi-specific.scss

```diff
@@ -1,3 +1,7 @@
+@use '../../node_modules/@cds/core/tokens/tokens';
```

Comment:
If this was in the lib code I'd say to use the `var(--token)` syntax instead - but it doesn't matter as much, it's only in the website code :)